test/parallel/test-whatwg-url-properties.js

```diff
@@ -25,6 +25,9 @@ const expected = ['toString',
                   'username', 'password', 'host', 'hostname', 'port',
                   'pathname', 'search', 'searchParams', 'hash', 'toJSON'];
 
+const obj = { toString() { throw new Error('toString'); } };
```

Comment:
I think these tests can be in `test-whatwg-url-setters.js`? This file is testing the property attributes are defined properly, the added tests are testing the setters are implemented properly.